syntax = "proto3";

service messageService {
  rpc message (instructionRequest) returns (messageResponse) {}
}

message instructionRequest {
  string estado = 1;
}

message messageResponse {
  repeated string respuesta = 1;
  repeated int32 usoCPU = 2;
}